sdcm/sct_config.py

```diff
@@ -1231,6 +1231,9 @@ class SCTConfiguration(dict):
         dict(name="perf_gradual_throttle_steps", env="SCT_PERF_GRADUAL_THROTTLE_STEPS", type=dict,
              help="Used for gradual performance test. Define throttle for load step in ops. Example: {'read': ['100000', '150000'], 'mixed': ['300']}"),
 
+        # StorageUtilizationTest
+        dict(name="action_type", env="SCT_ACTION_TYPE", type=str,
```

Comment:
This is to control test-script via yaml entries. We can have something like `storage_utitlization_scaleout.yaml` with `action_type` as scale_out and another yaml like `storage_utitlization_scalein.yaml` with `action_type` as scale_in and so on.  This can used to cover our existing testcases in testplan with single script and multiple yaml files.

Reply:
so everytime we add custom yaml parameter to some SCT test, we need to add it here?

Reply:
Yes, I think so. I ran into unknown config entry error without these changes on this file.

Reply:
yes, we are strict about the SCT configuration, every single one should be listed here and with explanations what's the usage of it.

if it's specific to some tests, or type of tests.

we need this to be able to convert correctly from environment variable to python types.
and for validation that the configuration (we cover it during SCT CI)

as @ Lakshmipathi  point multiple config files can be used, for example to have a set of configuration that can be shared between multiple tests. or for enabling/disabling features.